include/dbg_macro/dbg.h

```diff
@@ -229,7 +251,7 @@ bool pretty_print(std::ostream& stream, P* const& value) {
   return true;
 }
 
-template <size_t N>
```

Comment:
why was this changed back to size_t? Is that required for Windows compatibility?